contents/blog/how-to-create-personas.md

```diff
@@ -0,0 +1,175 @@
+| How frequently do these users set reminders or snooze emails?                                      |
+
+For example, our data may show that users who complain about email overload receive over 100 emails a day – an excellent detail to use in a user persona! 
+
+To help you brainstorm questions to ask for your own product, here's a list of insights to dig into:
+
+- What percentage of your active users match your user persona?
+- How often do they use your product?
+- What are their most used features?
+- Which features are rarely or never used?
+- Are there common drop-off points or areas of friction?
+- How do users from different job titles or roles use the product differently?
+
+> 💡 **PostHog Tip:** If you're using PostHog, you can use filters to create a [cohort](/docs/data/cohorts) of users in your user persona. Then, you can easily break down your insights using this cohort.
+
+### Step 4: Create a user persona with your findings
+
+The final step is to use what you've learned about your user and create a user persona. Summarize your findings into the three key categories we mentioned earlier (job summary, motivations, and frustrations).
+
+Make your summary detailed enough to show who your users are, but keep it short and simple so your team can understand it easily.
+
+To illustrate this with an example, below is the completed user persona for our example email app. We created this persona after learning that:
+
+- Our key users are senior executives.
+- Email overload is their biggest pain point
+- Prioritization is important to them.
+- People who complained about email overload were receiving 100 emails a day.
+
+![The user persona for our example email app](../images/blog/how-to-create-personas/email-app-example-persona.png)
+
+## Next steps
+
+Your user persona is the voice of your user. It should guide your product development and company strategy. 
+
+Incorporate them into team meetings and encourage your team to always ask, "How will this benefit our key user personas?". By doing so, you build a culture of empathy for your users, ensuring that you make the best products for them.
+
+For instance, at PostHog our product teams use our personas to create their team mission, values, and quarterly goals. Since our company handbook is public, you can view precisely how by visiting each team's page – e.g., our [Feature Success](/handbook/small-teams/feature-success#personas) and [Web Analytics](/handbook/small-teams/web-analytics#who-are-we-building-for) teams.
+
+## Frequently asked questions
+
+### How many user personas should I have?
+
+There are two things to remember:
+
+- The more personas you have, the greater the chance team members will not remember information about them.
+- Your user personas should focus on your most important user groups — those people you really want your product to reach.
+
+With this in mind, having only one user persona may be enough. You should have at most five, though.
```

Comment:
Would love feedback on this:

Is it strange to have an FAQ section with only one question?

The bit on " How many user personas should I have?" felt too long to put into another section. At the same time, I dont want to needlessly add filler questions to the FAQ

Reply:
It is a bit awkward, maybe as a quote callout after:

"If you're looking for more examples, GitLab has published the [16 different user personas they use](https://about.gitlab.com/handbook/product/personas/)."

I think this length (or a touch shorter) would be fine. Maybe cut this sentence down to:

- Your user personas should focus on those people you really want your product to reach.